routes.js

```diff
@@ -0,0 +1,63 @@
+const express = require('express')
+const router = express.Router()
+const app = express()
+const low = require('lowdb')
+const fileAsync = require('lowdb/lib/storages/file-async')
+const db = low('./db/db.json', {
+  storage: fileAsync
+})
+
+app.use(express.static('./public'))
+
+router.get('/movies', (req, res) => {
```

Comment:
Because all of the router routes are prepended with `/movies`, it would be good to do `app.use("/movies", router);` in `app.js` and `router.get("/", ...` here.